### PR TITLE
Fix upstream changes, pin commit for aiocoap

### PR DESCRIPTION
**Pull request by lwis**

### CODE REVIEW

File: pytradfri/api/aiocoap_api.py [hunk 1]
```diff
@@ -27,10 +27,10 @@ def _get_psk(self, host, port):
 
 
 def _patched_datagram_received(self, data, addr):
-    self._dtls_socket.handleMessage(self._connection, data, 0)
+    self.parent._dtls_socket.handleMessage(self.parent._connection, data, 0)
```
Comment (balloob) [inline pytradfri/api/aiocoap_api.py]: Should we see to get this fixed upstream?

**Comment by balloob** [comment]
I'm on the move right now so can't do a release.

**Comment by ggravlingen** [comment]
I released the change and pushed 2.2.2 to pypi @balloob @lwis

**Comment by lwis** [comment]
@ggravlingen updated in HA :+1:

**Pull request merged**